drivers/metadata/postgres/metadata.go

```diff
@@ -112,7 +113,7 @@ func (r metaReader) Tables(f metadata.Filter) (*metadata.TableSet, error) {
 FROM pg_catalog.pg_class c
      LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
 `
-	conds := []string{"n.nspname !~ '^pg_toast' AND c.relkind != 'c'"}
+	conds := []string{"n.nspname !~ '^pg_toast' AND c.relkind IN ('r', 'p', 's', 'f', 'v', 'm', 'S')"}
```

Comment:
Can you add a comment mentioning that `c` is not included? This is the only remaining type. It might make it easier to debug things in future.

Reply:
done